server/src/controllers/chess-game.js

```diff
@@ -1,11 +1,26 @@
 import CGUtil from '../utils/chess-game-util';
+import User from '../models/user';
 
 const getRating = (req, res) => {
   const { playerOneRating, playerTwoRating, winner } = req.query;
   const newRating = CGUtil.calcRating(playerOneRating, playerTwoRating, winner);
   res.json(newRating);
 };
 
+const leaderboard = (req, res) => {
+  const query = User.find();
```

Comment:
is this lazy evaluation? or all users in the db are loaded to memory soon we execute this?
Please avoid if not lazy, it will quickly eat up memory.